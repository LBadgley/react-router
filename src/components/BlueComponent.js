import React from 'react';

export default function BlueComponent() {
  const splotchStyle = {
    width: '550px',
    height: '550px',
    backgroundColor:'blue',
    borderRadius: '100%'
  };
  return <div style={splotchStyle}></div>;
}
