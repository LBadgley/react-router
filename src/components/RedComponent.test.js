import React from 'react';
import { shallow } from 'enzyme';
import BlueComponent from './BlueComponent';

describe('RedComponent Test', () => {
  it('renders the red component', () => {
    const wrapper = shallow(<BlueComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
