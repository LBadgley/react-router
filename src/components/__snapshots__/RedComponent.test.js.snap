// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RedComponent Test renders the red component 1`] = `
<div
  style={
    Object {
      "backgroundColor": "blue",
      "borderRadius": "100%",
      "height": "550px",
      "width": "550px",
    }
  }
/>
`;
